haystack/components/rankers/__init__.py

```diff
@@ -4,12 +4,14 @@
 
 from haystack.components.rankers.lost_in_the_middle import LostInTheMiddleRanker
 from haystack.components.rankers.meta_field import MetaFieldRanker
+from haystack.components.rankers.meta_field_grouper_ranker import MetaFieldGroupingRanker
```

Comment:
```suggestion
from haystack.components.rankers.meta_field_grouping_ranker import MetaFieldGroupingRanker
```

Let's change the name of the file.